src/renderer/lib/hooks/useAppUpdate.ts

```diff
@@ -1,8 +1,16 @@
 import { useToasts } from "@/lib/hooks/useToasts";
 
-export const useInstallAppUpdate = () => {
+export const useAppUpdate = () => {
   const { showError } = useToasts();
 
+  const checkForAppUpdates = async () => {
```

Comment:
```suggestion
  const checkForAppUpdates = React.useCallback(async () => {
```
Do you mind wrapping both of these in `useCallback` for me?